.github/workflows/release.yml

```diff
@@ -0,0 +1,79 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      name:
+        description: 'Release name'
+        required: true
+        default: 'v0.0.0'
+      is-prerelease:
+        description: 'Is this a pre-release?'
+        required: true
+        type: boolean
+        default: true
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    container: fedora:39
+    permissions:
+      contents: write
+    steps:
+      # Dependencies must be installed before the checkout step, otherwise the
+      # .git directory will be missing and tag creation will fail.
+      - name: Install Dependencies
+        run: |
+          echo "Installing workflow dependencies"
+          dnf install -y git
+          echo "Installing build dependencies"
+          dnf install -y \
+            bash coreutils crun genisoimage \
+            libselinux-devel libvirt-client \
+            libvirt-daemon-driver-qemu \
+            libvirt-daemon-log qemu-img \
+            qemu-system-x86-core \
+            shadow-utils util-linux \
+            virtiofsd
```

Comment:
These packages aren't actually needed to build crun-vm.

Reply:
[d0fa75f](https://github.com/containers/crun-vm/pull/7/commits/d0fa75ff384d61738d9c44a031704e794ad2355b)